webserver/templates/user/export.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html' %}
+{% block title %} Export for "{{ user.musicbrainz_id }}" - ListenBrainz {% endblock %}
+
+{% block content %}
+	{% if current_user.is_authenticated() and current_user == user %}
```

Comment:
Use indentation consistently with other files in the project. For HTML templates it's 2 spaces, not tabs.